spec/support/shared_examples/http_request_methods.rb

```diff
@@ -0,0 +1,29 @@
+shared_examples 'available http request methods' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.